src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Rectangle extends FigCol {
+    private double width;
+    private double height;
+
+    public Rectangle(double width, double height, Color color) {
```

Comment:
Passing expressions directly in the constructor is not recommended for better code readability. Consider creating separate variables for 'width' and 'height' before passing them to the constructor.